bin/garden-config

```diff
@@ -29,8 +29,6 @@ indent() { sed 's/^/  /'; }
 targetDir="${1:-}"; shift || eusage 'missing target-dir'
 [ -n "$targetDir" ]
 
-aptVersion="$("$thisDir/.apt-version.sh" "$targetDir")"
```

Comment:
Since `.apt-version.sh` is not used anymore, shouldn't we also delete this script then?